apyros/metalog.py

```diff
@@ -38,7 +46,13 @@ def __init__( self, filename=None ):
         else:
             self.replay = True
             self.filename = filename
-            self.lines = open( self.filename ).readlines()
+            if filename.endswith('.zip'):
+                self.zf = zipfile.ZipFile(filename)
+                tmp_meta = [info.filename for info in self.zf.infolist() if 'meta_' in info.filename]
+                assert len(tmp_meta) == 1, tmp_meta
+                self.lines = self.zf.read(tmp_meta[0]).split('\n')  # workaround for iterator
```

Comment:
unfortunately no - I already have the string, and not file ... maybe change to ZipFile.open().readlines()??